api/test/e2e/admin-regions/admin-regions-get-trees.spec.ts

```diff
@@ -263,10 +263,10 @@ describe('AdminRegions - Get trees', () => {
         .set('Authorization', `Bearer ${jwtToken}`)
         .expect(HttpStatus.OK);
 
-      expect(response.body.data[0].id).toEqual(parentWithNoChildAdminRegion.id);
-      expect(response.body.data[0].attributes.children).toEqual([]);
-      expect(response.body.data[1].id).toEqual(parentAdminRegion.id);
-      expect(response.body.data[1].attributes.children[0].id).toEqual(
+      expect(response.body.data[1].id).toEqual(parentWithNoChildAdminRegion.id);
```

Comment:
this feels like a test depending on a specific order of elements, which typically it should not unless the api endpoint makes a commitment around the order in which it returns the results

Reply:
Yes, this is enforced by the orderBy you pointed out above